### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Wednesday, 12th of June 2024 2:30pm UTC

### DISCUSSION THREAD

**Discussion by t-lo** [flatcar/Flatcar#1443] [Flatcar Office Hours]
In the office hours we discuss news and happenings in and around Flatcar, provide a space for our user community to discuss with maintainers, and do our release planning.

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording:  https://www.youtube.com/watch?v=i04yEYnDujg
  - All recordings: https://www.youtube.com/@FlatcarLinux/live
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

## Agenda items

- @nesv and @srust from Akamai - talk about the support for Flatcar on Linode and do a short demo
  - https://github.com/flatcar/flatcar-website/pull/327 (WIP documentation)
  - demo sources: https://github.com/nesv/flatcar-demo

### Release Planning

- https://github.com/orgs/flatcar/projects/7/views/8

### Administrative tasks after meeting concluded
- [x] Create a new Github discussion for the next Flatcar Office Hours: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-office-hours
- [x] Close this discussion as "Resolved".

**Comment by srust** [comment]
@nesv and @srust from Akamai can come and talk about the support for Flatcar on Linode and do a short demo. Thanks!

**Comment by srust** [comment]
If you want to try out flatcar on Linode, there is a $100 free credit for new accounts: https://www.linode.com/lp/free-credit-100/

**Comment by nesv** [comment]
Here is a link to the repository with the Makefile and other things I used for the demo: https://github.com/nesv/flatcar-demo
